CMakeLists.txt

```diff
@@ -358,21 +424,21 @@ endif()
 # The library is installed by src/CMakeLists.txt
 #--------------------------------------------------------------------
 if (GLFW_INSTALL)
-    install(DIRECTORY include/GLFW DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
+    install(DIRECTORY include/GLFW DESTINATION "${GLFW_INSTALL_INCLUDEDIR}"
             FILES_MATCHING PATTERN glfw3.h PATTERN glfw3native.h)
 
     install(FILES "${GLFW_BINARY_DIR}/src/glfw3Config.cmake"
                   "${GLFW_BINARY_DIR}/src/glfw3ConfigVersion.cmake"
-            DESTINATION "${GLFW_CONFIG_PATH}")
+            DESTINATION "${GLFW_INSTALL_CONFIGDIR}")
 
     install(EXPORT glfwTargets FILE glfw3Targets.cmake
             EXPORT_LINK_INTERFACE_LIBRARIES
-            DESTINATION "${GLFW_CONFIG_PATH}")
+            DESTINATION "${GLFW_INSTALL_CONFIGDIR}")
```

Comment:
Would you be interested in installing to a namespace?

```diff
 install(EXPORT glfwTargets FILE glfw3Targets.cmake
+        NAMESPACE glfw3::
         EXPORT_LINK_INTERFACE_LIBRARIES
         DESTINATION "${GLFW_INSTALL_CONFIGDIR}")
```

This is a relatively common practice these days (I believe), it makes it so that `find_package(glfw3)` users can `target_link_libraries(glfw3::glfw)`.  Namespaces really shine when there are multiple targets being installed, but even for single-library installs it's pretty common.  Happy to add if you like it, but we may want to add an alias library to regular `glfw` to support older projects, or even better, make this a prominent marker of `3.3` release -- `glfw3::glfw` is the new way?  Hehe